device_calendar/android/src/main/kotlin/com/builttoroam/devicecalendar/CalendarService.kt

```diff
@@ -34,35 +34,30 @@ import android.provider.CalendarContract.Events
 import android.content.ContentValues
 import com.builttoroam.devicecalendar.common.Constants.Companion.EVENT_PROJECTION_END_DATE_INDEX
 import com.builttoroam.devicecalendar.common.Constants.Companion.EVENT_PROJECTION_START_DATE_INDEX
+import com.builttoroam.devicecalendar.common.ErrorCodes.Companion.DELETING_RECURRING_EVENT_NOT_SUPPORTED
 import com.builttoroam.devicecalendar.common.ErrorCodes.Companion.EVENTS_RETRIEVAL_FAILURE
 import com.builttoroam.devicecalendar.common.ErrorCodes.Companion.EVENT_CREATION_FAILURE
 import com.builttoroam.devicecalendar.common.ErrorMessages.Companion.CREATE_EVENT_ARGUMENTS_NOT_VALID_MESSAGE
+import com.builttoroam.devicecalendar.common.ErrorMessages.Companion.DELETING_RECURRING_EVENT_NOT_SUPPORTED_MESSAGE
 import com.builttoroam.devicecalendar.common.ErrorMessages.Companion.EVENTS_START_DATE_LARGER_THAN_END_DATE_MESSAGE
+import com.builttoroam.devicecalendar.models.CalendarMethodsParametersCacheModel
 import java.util.*
 
 
 public class CalendarService : PluginRegistry.RequestPermissionsResultListener {
```

Comment:
I think it's worth renaming this class to be CalendarDelegate similar to the image picker plugin as a service has a different meaning in Android https://developer.android.com/guide/components/services

Reply:
Yeah, good point - don't like how it works in Android though

Reply:
i know you don't :)